packages/ui/src/components/button.tsx

```diff
@@ -18,6 +18,7 @@ const buttonVariants = cva(
       },
       size: {
         default: 'h-10 py-2 px-4',
+        xs: 'h-6 px-2 rounded-full text-xs', // pill-shaped
```

Comment:
this is the 🍖 and 🥔s of the change, the rest is basically usage and re-ordering elements